currencyConverter.js

```diff
@@ -0,0 +1,52 @@
+var request = require('request');
+var cheerio = require('cheerio');
+
+/**
+* Rounds off currency output 2 decimal places
+* @param currency is the result returned from sendRequest
+*/
+var roundOff = function (currency) {
+
+	var result = Math.round(currency * 100) / 100;
+	return result.toFixed(2);
+
```

Comment:
remove this blank line and add one above the `return`. Typically you want to have a blank line above returns. Just a common style.